cassandane/tiny-tests/JMAPContacts/card_set_onlineservices

```diff
@@ -0,0 +1,303 @@
+#!perl
+use Cassandane::Tiny;
+
+use Digest::MD5 qw(md5_hex);
+
+my sub install_test {
+    my ( $action, $name, $arg ) = @_;
```

Comment:
as of yesterday you can instead write:

```perl
my sub install_test ($action, $name, $arg) {
```

Note that this enforces strict arity:  Without a default, an argument is mandatory.